src/FakeItEasy/Configuration/RuleBuilder.cs

```diff
@@ -164,6 +260,38 @@ public IAnyCallConfigurationWithReturnTypeSpecified<TMember> Where(Func<IFakeObj
                 this.ParentConfiguration.RuleBeingBuilt.ApplyWherePredicate(predicate, descriptionWriter);
                 return this;
             }
+
+            public virtual IAfterCallSpecifiedConfiguration<IReturnValueConfiguration<TMember>> AssignsOutAndRefParametersLazily(Func<IFakeObjectCall, ICollection<object>> valueProducer)

```

Comment:
> (Likewise none of the RuleBuilder methods need to be virtual, but that was the case even before this PR showed up, so maybe I digress.)

I can change it, but maybe it would be better to do it in a separate PR, since it's unrelated to the changes in this PR.